src/client/Microsoft.Identity.Client/AuthScheme/IAuthenticationScheme.cs

```diff
@@ -9,10 +9,10 @@ namespace Microsoft.Identity.Client.AuthScheme
     /// <summary>
     /// Used to modify the experience depending on the type of token asked. 
     /// </summary>
-    internal interface IAuthenticationScheme
+    public interface IAuthenticationScheme
```

Comment:
This is internal on the preview that was pushed. Can the namespace also have `AuthScheme` removed?

Reply:
Also, I don't see release notes for the current preview, so I cannot backtrack the release to a commit hash. Please publish preview release notes so customers can more easily track where the release comes from. Thanks.

Reply:
It's public in IDDP

Reply:
@jmprieur @jennyf19 - have we decided on a name? `AuthenticationProtocol` ?

Reply:
We typically dont publish release notes for prototypes because things can change so unexpectedly but I can certainly add them.